### PR TITLE
#544 Run emotion codemod

### PR DESCRIPTION
**Pull request by champagneabuelo**
Adds new linting rules for emotion, and adds a file to .gitignore for Intellij IDEs.

### CODE REVIEW

**Comment by champagneabuelo** [comment]
Not _quite_ sure if I'm doing this correctly, but I was able to run the linter from the project's base directory and get all the relevant emotion linting feedback...

**Comment by jaronheard** [comment]
> Not quite sure if I'm doing this correctly, but I was able to run the linter from the project's base directory and get all the relevant emotion linting feedback...

Sounds like you're on the right track 🙌

**Comment by champagneabuelo** [comment]
After reviewing the requirements for this ticket, I believe it's ready for review!

File: .eslintrc [hunk 1]
```diff
@@ -4,8 +4,12 @@
   "env": {
     "mocha": true
   },
-  "plugins": ["react", "prettier"],
+  "plugins": ["react", "prettier", "emotion"],
   "rules": {
+    "emotion/jsx-import": "error",
+    "emotion/no-vanilla": "error",
```
Comment (jaronheard) [inline .eslintrc]:
```suggestion
    "emotion/no-vanilla": "warn",
```

**Pull request merged**